### DISCUSSION TITLE
[Off Topic] Almost an accident with a wild boar

### DISCUSSION THREAD

**Discussion by wolfgang-ch** [mytourbook/mytourbook#1093] [General]
Yesterday: The route is clear and it goes through the forest with a descent of about 8%, so you drive faster.

At first, a car starts to overtake me, it was completely in the other lane because at that speed I drive pretty much in the middle. So far so good, but suddenly 2 wild boars cross the road from left to right, I brake, the car breaks hard which was already slightly ahead of me and then a 3rd wild boar comes when I and the car were about at the same height, but the wild boar probably turns away because I didn't hear any impact and the car then drove on but then stayed behind me because soon after that you had to brake hard because of a junction with another street.

Without the car I probably would have collided with the 3rd wild boar because you can't brake down the mountain so quickly with a bicycle or it could also be that without the car the wild boars might not have crossed the road so quickly. 

It all happened so fast, that's why I didn't remember how I had braked, but according to the graphics, I had already braked hard.

It's some surreal that an overtacing car can protect your.

![image](https://user-images.githubusercontent.com/1283445/236659948-bab56589-6ec4-4c0e-af9b-bc2fedb5e2df.png)

**Comment by FJBDev** [comment]
That's a scary story!
Do you wear a camera on your bike while you ride ? That would have been great footage!

I love wild boars though..... with mustard 😄

**Reply by wolfgang-ch** [reply]
I don't have an action cam which is recording the tour, I also think there are no cameras which could record several hours with an accumulator, this happend in the third hour of the tour.

This was roughly the location where it happened

![image](https://github.com/mytourbook/mytourbook/assets/1283445/11c76b3e-8e3a-4f72-9f40-8dcaaa4c3220)
